### DISCUSSION TITLE
Bluetooth between device app and linux, without zepp app

### DISCUSSION THREAD

**Discussion by aptinov** [zepp-health/zeppos-forum#224] [Q&A]
Hi,
Is it possible to have bluetooth communication between a device app and a linux machine, to get for instance heart rate, without using zepp app ?

**Comment by MyoungXUE** [comment]
Hi, the latest version of Zepp OS 2.1 doesn't support this feature yet, you have to use the Zepp App to communicate with the outside world, please refer to https://docs.zepp.com/docs/guides/architecture/arc/#communication- for more details on how to communicate relationships.

In Zepp OS 3.0, we have planned a Bluetooth-related feature that allows direct connection between the watch and other devices. For more information, please refer to our preview documentation at https://docs-testing.zepp.com/docs/next/reference/device-app-api/newAPI/ble/createConnect/
